src/components/MonitorList.vue

```diff
@@ -95,11 +95,22 @@ export default {
             disableSelectAllWatcher: false,
             selectedMonitors: {},
             windowTop: 0,
-            filterState: {
-                status: null,
-                active: null,
-                tags: null,
-            }
+            statusStates: {
```

Comment:
I think you added this as a sort of indirecton between the query parameters and the state.
I think you are overcomplicating this a bit, right?

PS: In any case, since this is not mutable state (but rather a constant LUT) I think this should not live here ^^

Reply:
Yes. The reason I did that is so URLs like `/dashboard?status=up` can be used - which is quite nice for user experience, like the use cases of guys at #3672. This means that IDs are not used in the URLs - the names are - so, the access is done through the names.

Do you want me to move it outside and make it a const?

Reply:
Using ids in the url is fine by me if the benefit is having less confusing code.

I think in this case maintainability outweighs having "prettier" url-query parameters => lets keep it simple